data/names/ru.json

```diff
@@ -3447,31 +3535,50 @@
       "Винсон",
       "Винтер",
       "Витале",
-      "Виттен",
       "Витт",
+      "Виттен",
       "Вишневски",
+      "Вишневский",
+      "Вмятина",
+      "Во",
       "Воган",
+      "Волк",
       "Вольф",
-      "Вонг",
+      "Воля",
```

Comment:
This is, more likely than not, a translation of the name Will as a common noun ("воля" as in "willpower"). It would be transcribed Уилл. (The only person with the name [Volya](https://en.wikipedia.org/wiki/Volya) on Wikipedia is [Pavel Volya](https://en.wikipedia.org/wiki/Pavel_Volya), a Russian comedian and TV show host.)

Reply:
Volya is shortened Vladimir, as seen in the book [Old Khottabych](https://en.wikipedia.org/wiki/Old_Khottabych). The main character is called Vol'ka or Volya(Волька, Воля). This shortened form is never used nowadays, though.